packages/api/models/collections.js

```diff
@@ -32,52 +32,95 @@ class Collection extends Manager {
     item.files.forEach((file) => checkRegex(file.regex, file.sampleFileName));
   }
 
+  /**
+   * Creates a new Collection model for managing storage and retrieval of
```

Comment:
I don't have enough insight into the code here regarding how we might refactor things for clarity, but I thought I should at least document the behavior that is otherwise not obvious, and requires looking at the implementation to figure this out.